packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -32,40 +38,86 @@ export interface IOutboxParameters {
 }
 
 export class Outbox {
+	private readonly mc: MonitoringContext;
 	private readonly attachFlowBatch: BatchManager;
 	private readonly mainBatch: BatchManager;
-	private readonly defaultAttachFlowSoftLimitInBytes = 64 * 1024;
+	private readonly defaultAttachFlowSoftLimitInBytes = 320 * 1024;
+
+	/**
+	 * Track the number of ops which were detected to have a mismatched
+	 * reference sequence number, in order to self-throttle the telemetry events.
+	 *
+	 * This should be removed as part of ADO:2322
+	 */
+	private readonly maxMismatchedOpsToReport = 3;
+	private mismatchedOpsReported = 0;
```

Comment:
You can get away with just one counter that decrements to zero

Reply:
I know, but I want the `x` out of `y` in the logs. We could deduce that with crafting a better kusto query but I figured the bundle size hit is minimal.